0-common/igluctl/.gitignore

```diff
@@ -12,3 +12,5 @@ lib_managed/
 src_managed/
 project/boot/
 project/plugins/project/
+
+.idea
```

Comment:
This is specific to a single IDE (IntelliJ) with a single project style (one that generates an .idea folder). I'd remove this seeing as the user can set it in their global gitignore if they want to.